src/derivatives/atlas.md

```diff
@@ -293,14 +288,14 @@ A guide for using macros can be found at
    "colin27-pipeline": {
       "sub-01": {
          "anat": {
-            "sub-01_atlas-AAL_dseg.json": "",
-            "sub-01_atlas-AAL_dseg.nii.gz": "",
-            "sub-01_atlas-AAL_dseg.tsv": "",
-            "sub-01_atlas-AAL_probseg.nii.gz": "",
-            "sub-01_label-brain_mask.nii.gz": "",
-            "sub-01_label-head_mask.nii.gz": "",
-            "sub-01_T1w.nii.gz": "",
-            "sub-01_T1w.json": "",
+            "atlas-AAL_dseg.json": "",
```

Comment:
The sub-01 part should remain in the filename, to indicate that it belongs to sub-01

Reply:
I agree with Martin on this it's too easy to overwrite other subject specific files without the `sub-<label>` remaining. Was this changed to make the sharing of a single subject atlas/template consistent with a multi-subject atlas/template?

Reply:
Reverted.